### DISCUSSION TITLE
Rocky Linux installation

### DISCUSSION THREAD

**Discussion by mmol67** [Xpra-org/xpra#3925] [General]
Hello!. I'm just installed Rocky Linux (Workstation Lite image), and I've noticed that you need to enable epel and cbr to get Xpra installed, or you will miss packages and libraries (like libturbojpeg.so)

As per https://wiki.rockylinux.org/rocky/repo/#community-approved-repositories , I did
```
dnf install epel-release
crb enable
```
And then Xpra installs ok. JFYI

**Comment by totaam** [comment]
I've made it clearer: https://github.com/Xpra-org/xpra/wiki/Download/_compare/be2b0a3b8d61af0993da803b2c44bde68e588ad2
